### PR TITLE
Closes #89: Setup Frontend README

### LINKED ISSUE #89: Setup Frontend README
**Is your feature request related to a problem? Please describe.**
As a contributor to this repository, I want to read a README, so that I can have an understanding of how to run the app and what to install

**Describe the solution you'd like**
A README file that explains what to install and how to build and make changes to the app

**Describe alternatives you've considered**
No README, this would mean people using the repository wouldn't know what to install or how to start and build the app

**Additional context**
None.

### PR DESCRIPTION
**Pull request by AngelaLH**
Closes #89 

Frontend README set up containing:
- Prerequisites to install
- Getting started
- Available scripts
- Learn more

### CODE REVIEW

File: telos-frontend/README.md [hunk 1]
```diff
@@ -1,45 +1,56 @@
...
+    - [npm](https://www.npmjs.com/get-npm)
+    - [Node.js](https://nodejs.org/en/)
+    - [Prettier](https://prettier.io/docs/en/install.html)
+    - [Lint](https://www.npmjs.com/package/lint)
 
-In the project directory, you can run:
+# Getting Started
 
-### `npm start`
+1. Open the command prompt and set the working directory as the `3Lancers` directory of your cloned repository. (`cd 3Lancers`)
```
Comment (RawiriHohepa) [inline telos-frontend/README.md]: can you change the cd command to `cd path/to/3Lancers` just to emphasise that they are navigating to the right folder?

File: telos-frontend/README.md [hunk 2]
```diff
@@ -1,45 +1,56 @@
...
+2. Run `npm install` to install all the required dependencies
```
Comment (RawiriHohepa) [inline telos-frontend/README.md:14]: Are we using `npm install` or `npm ci`? I'm pretty sure either one is fine as long as we keep it consistent among everyone - that may be what was causing the really big package-lock diffs

**Review by RawiriHohepa — CHANGES_REQUESTED** [review]
Looks good! Just a small change and a decision for the team to make as a whole over whether to use `npm install` or `npm ci` - happy to approve once those are sorted!

File: telos-frontend/README.md [hunk 3]
```diff
@@ -1,45 +1,56 @@
...
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+# Prerequisites
+- You have already forked the repository and cloned your fork to your local machine. More information [here](https://gist.github.com/Chaser324/ce0505fbed06b947d962)
+- You have the following installed:
+    - [npm](https://www.npmjs.com/get-npm)
+    - [Node.js](https://nodejs.org/en/)
+    - [Prettier](https://prettier.io/docs/en/install.html)
+    - [Lint](https://www.npmjs.com/package/lint)
```
Comment (ambiyu) [inline telos-frontend/README.md]: Should be ESLint

File: telos-frontend/README.md (hunk 2 above)
Comment (RawiriHohepa) [reply telos-frontend/README.md:14]: Based on offline discussion it seems that we'll stick with npm install

**Review by RawiriHohepa — APPROVED** [review]
Looks good now, thanks Angela!

**Review by ambiyu — APPROVED** [review]
LGTM

**Review by milkyway404 — APPROVED** [review]

**Pull request merged**